test/test_cuda.py

```diff
@@ -1705,7 +1705,7 @@ def _spawn_method(self, method, arg):
         with ctx.Pool(1, initializer=self.mute) as pool:
             errors = pool.map(method, [arg])
             for e in errors:
-                if 'device-side assert triggered' not in str(e):
+                if 'self contains either `inf`, `nan` or element < 0' not in str(e):
```

Comment:
You can leave the error message as before, and test multinomial with replacement. This test serves double duty - asserts that multinomial implementation is correct, and that asserts are properly triggered in cuda kernels. With this change, the second goal is not served.

Reply:
Makes sense.